docs/read_the_docs/source/workflow_inputs/characteristics.rst

```diff
@@ -40097,6 +40097,12 @@ Arguments
      - Double
      -
      - The maximum capacity limit applied to the auto-sizing methodology. If not provided, no limit is used.
+   * - ``heating_system_2_fraction_heat_load_served``
```

Comment:
This argument is being added here under "HVAC Secondary Heating Efficiency" because you've added new "Fuel Wall/Floor Furnace" options which assign `heating_system_2_fraction_heat_load_served`. But that argument is already assigned in "HVAC Secondary Heating Partial Space Conditioning"...

Reply:
I think this will fix this copy error. [3ccb1cd](https://github.com/NREL/resstock/pull/1329/commits/3ccb1cd000da769878d212e17b8bbea6ae587a84)